apps/video/js/video.js

```diff
@@ -1313,10 +1310,12 @@ function handleSliderTouchEnd(event) {
 
   dom.playHead.classList.remove('active');
 
+  // We didn't move the play head while the user's finger was on it
+  // so now it may be in the wrong place.
+  updateVideoControlSlider();
```

Comment:
The issue isn't related to seeker. It is that we don't update the play head position on timeupdate events when the user's finger is down. Now that we're playing during the drag, it is possible for the play head to get behind the actual time.  And it is a serious bug if the user holds their finger down until the video stops. Then the play head is stuck at the wrong place.  

See the `if (!dragging)` statement in updateVideoControlSlider().

So yes, this is still needed.